### PR TITLE
JSTL/Tags: Port Import Flushing Fix

### LINKED ISSUE #18631: Port Fix for ee4j jstl api issue #153
Needed for a dynacache test

https://github.com/eclipse-ee4j/jstl-api/issues/153

### PR DESCRIPTION
**Pull request by volosied**
Fixes #18631

### CODE REVIEW

**Comment by volosied** [comment]
#build

File: dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSTLTests.java [hunk 1]
```diff
@@ -183,4 +190,41 @@ public void testFormatTags() throws Exception {
...
+        WebRequest request = new GetMethodWebRequest(url);
+        WebResponse response = wc.getResponse(request);
+
+        String text =  response.getText();
+        LOG.info("Servlet response : " + response.getText());
+
+        int beginIndex = text.indexOf("Begin"); // 1006
+        int endIndex = text.indexOf("End"); // 1014
+
+        assertTrue("Something went wrong with the flushing within the import tag", 8 == (endIndex - beginIndex));
```
Comment (volosied) [inline dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSTLTests.java:227]: Checking for a multi line string proved difficult, so I looked at the indices instead.

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#build

**Review by pnicolucci — APPROVED** [review]

**Comment by volosied** [comment]
#run-libby-bot

**Pull request merged**